test/unit-test/stun_deserializer/stun_deserializer_utest.c

```diff
@@ -254,66 +254,66 @@ void test_StunDeserializer_GetNextAttribute_IceControlled( void )
-                    TEST_ASSERT_EQUAL( 0x1234567890ABCDEF,
-                                       iceControlled );
-                }
-                break;
-
-                default:
-                {
-                    TEST_FAIL_MESSAGE( "Unexpected attribute type!" );
-                }
-                break;
+            case STUN_ATTRIBUTE_TYPE_ERROR_CODE:
+            {
+                result = StunDeserializer_ParseAttributeErrorCode( &( attribute ),
+                                                                   &( errorCode ),
+                                                                   &( errorPhrase ),
+                                                                   &( errorPhraseLength ) );
+                TEST_ASSERT_EQUAL( STUN_RESULT_OK,
+                                   result );
+                TEST_ASSERT_EQUAL( 600,
+                                   errorCode );
+                TEST_ASSERT_EQUAL_STRING_LEN( "Error Phrase",
+                                              &( errorPhrase [ 0 ] ),
+                                              errorPhraseLength );
+            }
+            break;
+
+            case STUN_ATTRIBUTE_TYPE_CHANNEL_NUMBER:
+            {
+                result = StunDeserializer_ParseAttributeChannelNumber( &( ctx ),
+                                                                       &( attribute ),
+                                                                       &( channelNumber ) );
+
+                TEST_ASSERT_EQUAL( STUN_RESULT_OK,
+                                   result );
+                TEST_ASSERT_EQUAL( 0x1234,
+                                   channelNumber );
+            }
+            break;
+
+            case STUN_ATTRIBUTE_TYPE_LIFETIME:
+            {
+                result = StunDeserializer_ParseAttributeLifetime( &( ctx ),
+                                                                  &( attribute ),
+                                                                  &( lifetime ) );
+
+                TEST_ASSERT_EQUAL( STUN_RESULT_OK,
+                                   result );
+                TEST_ASSERT_EQUAL( 0x0000EA60,
+                                   lifetime );
+            }
+            break;
+
+            case STUN_ATTRIBUTE_TYPE_ICE_CONTROLLED:
+            {
+                result = StunDeserializer_ParseAttributeIceControlled( &( ctx ),
+                                                                       &( attribute ),
+                                                                       &( iceControlled ) );
+
+                TEST_ASSERT_EQUAL( STUN_RESULT_OK,
+                                   result );
+                TEST_ASSERT_EQUAL( 0x1234567890ABCDEF,
+                                   iceControlled );
+            }
+            break;
+
+            default:
+            {
+                TEST_FAIL_MESSAGE( "Unexpected attribute type!" );
+            }
+            break;
```

Comment:
Could you udpate the uncrustify.cfg to align the indent for switch case?

Reply:
Yes, I have now updated the `indent_switch_case              = 4  ` for uncrustify.cfg and committed the changes.